components/RecipePreview/RecipePreview.tsx

```diff
@@ -0,0 +1,52 @@
+import styled from 'styled-components';
+import Image from 'next/image';
+
+export default function RecipePreview({ recipe }) {
+  const { title, imageUrl } = recipe;
+  return (
+    <StyledRecipePreview>
+      <ImageWrapper>
+        <Image
+          src={imageUrl}
+          alt={`picture of ${title}`}
+          quality={80}
+          fill
+          style={{ objectFit: 'cover' }}
```

Comment:
No inline styling please :)
It bad practice to combine Styled Components and inline styling :P

Reply:
Thanks, we changed it! ;)